Assets/Scripts/Dodo.cs

```diff
@@ -54,11 +54,9 @@ void FixedUpdate()
     {
         if (!isOnBridge)
         {
+            _behaviourTimer = Time.time % behaviourChangeSpeed;
             Move();
```

Comment:
@Cookiezznz is this OK? Or do we need to move while on the bridge?

Reply:
Or should behaviourTimer always be added?

Reply:
I think you're right - We shouldn't be moving if we're on the bridge - Because then he'd walk off. I think its fine to pause changing of behaviours until he is back off the bridge.